bench/workgen/workgen.cpp

```diff
@@ -305,6 +307,26 @@ get_dir_size_mb(const std::string &dir)
     return result / WT_MEGABYTE;
 }
 
+// 10 random characters + Null terminator
+#define DYNAMIC_TABLE_LEN 11
```

Comment:
Does it need to be this long? Using 5 characters gives almost one billion possible combinations.

Reply:
No, we don't need that long. Switched to 5.